packages/apps/graph/src/graph/mappers/signer-mapper.ts

```diff
@@ -0,0 +1,23 @@
+import type { Signer } from '@prisma/client';
```

Comment:
What do we need these mapper functions for exactly? Doesn't pothos do this already?

Reply:
Pothos doesnt because we map the capabilities to clist (capabilities list). However, I managed to couple signers with prisma again so I completely removed this mapper function